CODEOWNERS

```
@@ -1 +1,3 @@
 **/* @github/codeql-action-reviewers
+
+/python-setup/ @github/codeql-python
```

Perhaps we also want to be pinged here?

```suggestion
/python-setup/ @github/codeql-python @github/codeql-action-reviewers
```

---

Good catch. I had forgotten that these CODEOWNERS lines were not additive :+1: